JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -0,0 +1,12 @@
+The **Popup** UI component is a pop-up window that overlays the current view and displays specified content.
+
+This demo illustrates the following **Popup** use cases:    
+
+- **Specify the content**       
+Define the **Popup** content within the UI component's markup.   
+
+- **Set the title**   
+Use the [showTitle](Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#showTitle) property to show or hide the **Popup** title. If the **showTitle** value is set to **true**, define the **Popup** title within the [title](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#title) property.         
+
+- **Show and hide the Popup**   
+Use two-way binding to define the [visible](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#visible) property and manipulate its value directly. If the **Popup** should be hidden on clicks outside its bounds, enable the [closeOnOutsideClick](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#closeOnOutsideClick) property.
```

Comment:
I would explain the two-way binding in more detail. Then the outside clicks and overlay could be separate paragraphs. 

An obvious question about the overlay is if you can disable or customize it. I'm saying there that it's theme-dependent. That way I don't have to sound negative ("you cannot customize it"). But I'm not sure that this is actually the case :-) 

```suggestion
In this demo, we use two-way binding to define the [visible](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#visible) property:

- When you press the Details button to show the window, the code sets the bound variable to true and invokes the popup. 
- As we mentioned above, there are several ways to close the popup - title bar button, ESC key, or outside clicks. In response to these actions, the popup hides itself, updates its visibility  setting and the bound variable.  

To disable outside clicks as a way to close the popup, use the [closeOnOutsideClick](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#closeOnOutsideClick) property.

When the popup appears, it displays a semi-transparent layer over its container. The layer's appearance is predefined and depends on the currently applied theme. 
```